src/isomorphic/modern/class/ReactBaseClasses.js

```diff
@@ -21,7 +21,28 @@ var warning = require('fbjs/lib/warning');
 /**
  * Base class helpers for the updating state of a component.
  */
+
+const addUpdaterMutationWarn = context => {
```

Comment:
I guess we still need an empty func that would not get stripped out right? So that it can be statically analyzed?

```
let addUpdaterMutationWarn = () => {};

if (__DEV__) {
  addUpdaterMutationWarn = context => {
  // ...
  // ...
```